packages/react-native/Libraries/Text/TextInput/Singleline/RCTUITextField.mm

```diff
@@ -115,6 +115,37 @@ - (void)setSecureTextEntry:(BOOL)secureTextEntry
   self.attributedText = originalText;
 }
 
+- (void)setDisableKeyboardShortcuts:(BOOL)disableKeyboardShortcuts
+{
+  static BOOL isFirstRender = YES;
+  static NSArray<UIBarButtonItemGroup *> *initialValueLeadingBarButtonGroups;
+  static NSArray<UIBarButtonItemGroup *> *initialValueTrailingBarButtonGroups;
```

Comment:
This doesn't look too good. It can probably lead to issues if you render multiple text fields next to each other, as the static property will be shared across instances.

Reply:
Done - 210ba707897ef851fbf314a736ea174be1549571.